packages/components/src/menu-item/style.scss

```diff
@@ -24,13 +24,16 @@
 	&:focus:not(:disabled):not([aria-disabled="true"]) {
 		@include menu-style__focus;
 	}
+}
 
-	// Don't wrap text until viewport is beyond the mobile breakpoint.
```

Comment:
This change directly undoes the intention of #8756.

Guess it depends if we'd rather:

Wrapped|Unwrapped
---|---
![Wrapped](https://user-images.githubusercontent.com/1779930/47229290-600f3f80-d395-11e8-93f9-0776981e1289.png)|![Unwrapped](https://user-images.githubusercontent.com/1779930/47229268-51c12380-d395-11e8-9afd-d2562b657dc5.png)

Also, the comment is misleading, where the previous behavior had the opposite effect.

cc @jasmussen @tofumatt

Reply:
@alexislloyd thoughts on this?

Reply:
Maybe worth noting currently there _is_ a max-width of the menu (480px), after which the text simply becomes clipped.

![image](https://user-images.githubusercontent.com/1779930/47232737-8685a880-d39e-11e8-9839-5ad933d0ed81.png)

Reply:
I think wrapped is better here, as long as we're careful not to have the descriptions go over two lines with standard font sizes.

Reply:
I'm also ok with editing down the first description to "Access all tools in one place"

Reply:
We also have the option to enforce a max-width (with wrapping) on the description specifically, allowing both the main label and shortcut text grow unrestrained.